Patches/Alpha Animals/ThingDefs_Misc/AlphaAnimals_CE_Patch_Projectiles.xml

```diff
@@ -366,6 +366,18 @@
 							<speed>75</speed>
 						</projectile>
 					</value>
+				</li> -->
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/VFECore.ExpandableProjectileDef[defName="AA_LaserBullet"]/projectile</xpath>
+					<value>
+						<projectile Class="CombatExtended.ProjectilePropertiesCE">
+							<damageDef>Flame</damageDef>
+							<flyOverhead>false</flyOverhead>
+							<damageAmountBase>10</damageAmountBase>
+							<ai_IsIncendiary>true</ai_IsIncendiary>
+							<speed>20</speed>
+						</projectile>
+					</value>

```

Comment:
Dunno if that will work, it is not that simple. It uses special code from vanilla expanded framework.